region/brinstar/kraid.json

```
@@ -1667,6 +1572,39 @@
         {
           "from": 1,
           "to": [
+            {
+              "id": 1,
+              "strats": [
+                {
+                  "name": "Leave With Runway",
+                  "notable": false,
+                  "requires": [],
+                  "exitCondition": {
+                    "leaveWithRunway": {
+                      "length": 3,
+                      "openEnd": 1
+                    }
+                  }
+                },
+                {
+                  "name": "Leave Shinecharged",
+                  "notable": false,
+                  "requires": [
+                    {"canShineCharge": {
+                      "usedTiles": 33,
+                      "openEnd": 2
+                    }},
+                    "canShinechargeMovement"
+                  ],
+                  "exitCondition": {
+                    "leaveShinecharged": {
+                      "framesRemaining": 120
+                    }
+                  },
+                  "note": "We won't worry about obstacles as requirements here, because achieving blue suit will destroy them anyhow."
```

This is no longer true. There are some requirements to get over them and to kill them with speed now, to keep this out of `Basic`
```suggestion
                    "canShinechargeMovement",
                    {"obstaclesCleared": ["A"]}
                  ],
                  "exitCondition": {
                    "leaveShinecharged": {
                      "framesRemaining": 120
                    }
                  }
```